.github/label-actions.yml

```
@@ -438,4 +454,23 @@
     If you are a paying Mend.io customer, please tell your support or customer contact that this issue is important to you.
 
 
+    Thanks, the Renovate team
+
+'auto:reduce-complexity':
+  comment: >
+    Hi there,
+
+
+    This discussion is too complex, and we recommend you simplify it to increase chances of getting help and finding a solution.
+
+
+    For example, if you've included your _whole_ config, and it's quite complex, while your problem relates to just one specific part, consider removing the parts that are not relevant to the problem. The best way to do this is to usually create a minimal reproduction repository.
```

```suggestion
    For example, if you've pasted your _whole_ complex config, while your problem is about just one part, consider removing the parts that are not relevant to your problem. The best way to do this is to create a [minimal reproduction](https://github.com/renovatebot/renovate/blob/main/docs/development/minimal-reproductions.md).
```

Simplify a bit, and link to the minimal reproduction guide.